hack/ginkgo-e2e.sh

```diff
@@ -144,6 +144,8 @@ if [[ "${GINKGO_NO_COLOR}" == "y" ]]; then
   ginkgo_args+=("--noColor")
 fi
 
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-${KUBE_CONTAINER_RUNTIME:-}}
```

Comment:
does this actually get plumbed through from configure.sh?

Reply:
yes, yes it does.